mobsf/templates/auth/users.html

```diff
@@ -0,0 +1,124 @@
+{% extends "base/base_layout.html" %}
+{% load static %}
+{% block sidebar_option %}
+sidebar-collapse
+{% endblock %}
+{% block extra_css %}
+<link href="{% static "adminlte/plugins/sweetalert2/sweetalert2.min.css" %}" rel="stylesheet">
+{% endblock %}
+{% block content %}
+<div class="content-wrapper">
+   <div class="content-header">
+   </div>
+   <div class="container-fluid">
+      <div class="row">
+         <div class="col-lg-12">
+            <div class="card">
+               <div class="card-body">
+                  <div class="h-100 d-flex align-items-center justify-content-center">
+                     <div class="">
+                        <div class="login-logo">
+                           <a href="">User Management</a>  <a href="{% url 'create_user' %}" id="add_user" class="btn btn-success" role="button"><i class="fa fa-user"></i> Add User</a>
+                        </div>
+                        <div class="card">
+                           <div class="card-body login-card-body">
+                            {% if not users %}
+                            <h3>No Users Available</h3>
+                            {% else %}
+                            <div class="table-responsive">
+                                <table class="table table-responsive-md">
+                                    <thead>
+                                        <tr>
+                                            <th><strong>User</strong></th>
+                                            <th><strong>Email</strong></th>
+                                            <th><strong>Role</strong></th>
+                                            <th><strong>Manage</strong></th>
+                                        </tr>
+                                    </thead>
+                                    <tbody>
+                                        {% for user in users %}
+                                                <tr>
+                                                    <td>
+                                                            <div class="profile-photo">
+                                                                <img src="https://www.gravatar.com/avatar/{{ user.email | md5}}" width="30" class="img-fluid rounded-circle" alt="">
```

Comment:
This was to add some personalization to the user accounts. Email associated with `https://gravatar.com/` will have an image displayed here. To change the image, you need to change them at gravatar.com

Reply:
Anyone uses it? 🤣
I never heard of it, but maybe I'm in a bubble.
Anyhow I will either check there is actual an icon, or at least split it to another column but for now it just looks bad I think...

Reply:
Oh it's pretty [popular](https://www.wappalyzer.com/technologies/miscellaneous/gravatar/), useful for websites that want to avoid a profile picture add/edit/update feature. I will move it into a column so the UI looks alright.